andela/activo-web
# #157470184 Users should be able to log out

#### What does this PR do?
Create the log-out functionality for activo application.

#### Description of tasks to be completed?
- Create profile drop-down component
- Create an interface for the drop-down component
- Write styles for the component
- Write logic for logging out user

#### How should this be manually tested?
- After cloning the application, run yarn install to install the required dependencies for the application.
- The application should open on activo-dev.andela.com:3000.
- Log in with your Andela account. You should be re-directed to the dashboard page.
- Next to your displayed name, click on the caret. A drop-down should be displayed with a text of `LOG OUT`.
- Click on it. You should now be re-directed to the authentication page.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#157470184](https://www.pivotaltracker.com/story/show/157470184)

#### Screenshots
<img width="1280" alt="screen shot 2018-06-12 at 3 17 42 pm" src="https://user-images.githubusercontent.com/24798364/41304761-f859392c-6e68-11e8-9243-c9904bc8100c.png">

<img width="1280" alt="screen shot 2018-06-12 at 3 26 33 pm" src="https://user-images.githubusercontent.com/24798364/41304779-02018eb6-6e69-11e8-96e4-23410cc0294a.png">
